tests/automation_framework/data/work_order/work_order_two_workloadid.json

```diff
@@ -9,8 +9,8 @@
         "notifyUri": "notifyuri",
         "workOrderId": "",
         "workerId": "",
-        "workloadId": "echo-result",
-	"workloadId": "heart-disease-eval",
+        "workloadId": "heart-disease-eval",
```

Comment:
How does this test work? first workload_Id is heart_disease and the second one is echo_result and the parameters are for heart disease.